hrui/tests/phpunit/bootstrap.php

```diff
@@ -4,6 +4,14 @@
 ini_set('safe_mode', 0);
 eval(cv('php:boot --level=classloader', 'phpcode'));
 
+// Add other extensions to include path to allow use of CRM/* classes
+$civiHRRoot = realpath(__DIR__ . '/../../../');
+$extensionDirs = glob($civiHRRoot . '/*', GLOB_ONLYDIR);
+foreach ($extensionDirs as $extensionDir) {
+  $include_path = $extensionDir . PATH_SEPARATOR . get_include_path( );
```

Comment:
We will still need to include the hrcore directory so that I can use the fabricator classes. I think it might be nice to include this in all bootstrap files, instead of having to manually modify the `include_path` each time you want to reference a class from another extension in civihr.